spot_ros2_control/hardware/include/spot_ros2_control/visibility_control.h

```diff
@@ -0,0 +1,63 @@
+// File modified. Modifications Copyright (c) 2024 Boston Dynamics AI Institute LLC.
+// All rights reserved.
+
+// --------------------------------------------------------------
+// Copyright 2021 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* This header must be included by all rclcpp headers which declare symbols
+ * which are defined in the rclcpp library. When not building the rclcpp
+ * library, i.e. when using the headers in other package's code, the contents
+ * of this header change the visibility of certain symbols which the rclcpp
+ * library cannot have, but the consuming code must have inorder to link.
+ */
+
+// Source:
+// https://github.com/ros-controls/ros2_control_demos/blob/master/example_1/hardware/include/ros2_control_demo_example_1/visibility_control.h
+
+#ifndef SPOT_ROS2_CONTROL__VISIBILITY_CONTROL_H_
+#define SPOT_ROS2_CONTROL__VISIBILITY_CONTROL_H_
```

Comment:
@amessing-bdai should these lines also be updated to `#pragma once`? Will this affect the other blocks below (or should those be updated as well)? This was another boilerplate file I took from ros2 control demos

Reply:
Yes, this should and you just need to remove the endif at the end of the file. Nothing else changes

Reply:
ya I would include it here too

Reply:
ok thanks, added in latest commit